### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by thaytharma**
Link image to Eufemia Portal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,4 +28,4 @@ Find more information about how to contribute in [Eufemia Portal - Development](
 </a>
 
 ## Licence
-Go to [LICENSE](https://github.com/dnbexperience/eufemia/blob/main/LICENSE).
\ No newline at end of file
+Go to [LICENSE](https://github.com/dnbexperience/eufemia/blob/main/LICENSE).
```
Comment (tujoworker) [inline README.md:31]: Do you know why this got "removed" and "added"?

**Review by tujoworker — APPROVED** [review]
Nice! 👍

File: README.md (hunk 1 above)
Comment (thaytharma) [reply README.md:31]: Hmm, this has not been changed. Looks like GitHub is acting weird 😂

**Pull request merged**